docs/guides/rookieguide.md

```diff
@@ -1,7 +1,126 @@
 # Poradnik dla rekrutów
 
-## Wstęp
+## *Wstęp*
```

Comment:
Please no bolding of headers (`#`, `##` etc. are headers and don't need emphasis).
```suggestion
## Wstęp
```